src/stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,219 @@
+import { Meta } from '@storybook/addon-docs'
```

Comment:
I guess we can remove this `src/stories/` folder altogether and don't deal with `.mdx` files at all. We can always add later if we find a use case